src/app.ts

```diff
@@ -45,16 +127,26 @@ async function main(): Promise<void> {
 
   // Wait for pending promises to resolve
   await Promise.all(promises);
+}
 
-  (await getBrowser()).close();
-
-  console.log('=> Done');
+function isValidURL(url : string)  {
+  return url.includes(COURSE_URL)
 }
 
 /**
  * Run the main function
  */
 main().catch((e) => {
   console.error(e.message);
+  if(e instanceof SilentError) {
+    process.exit(0);
+  }
   process.exit(1);
 });
+
+//Custom Error to exit node process without showing error
+class SilentError extends Error {
```

Comment:
This was added because process was not exiting while returning. If @shihabmridha  you can make it work, we can remove it.